src/styles/index.scss

```diff
@@ -1,3 +1,237 @@
-body {
+.page {
   margin: 0;
+  box-sizing: border-box;
```

Comment:
The `box-sizing: border-box;` property should be applied to all elements, including pseudo-elements. You can use the universal selector `*` for this purpose. This will ensure that the padding and border of an element no longer increase its width.